### PR TITLE
front: fix relief on map

### LINKED ISSUE #9341: front: relief is always displayed
### What happened?

relief is always displayed by default, even though we set the relief coefficient to 0 by default.

### What did you expect to happen?

Relief should not be displayed by default

### How can we reproduce it (as minimally and precisely as possible)?

1. Open the stdcm map

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

.

### PR DESCRIPTION
**Pull request by clarani**
closes #9341

### CODE REVIEW

File: front/src/common/Map/Layers/HillShade.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { Source, type LayerProps } from 'react-map-gl/maplibre';
+
+import OrderedLayer from 'common/Map/Layers/OrderedLayer';
+import { useSelector } from 'react-redux';
+import { getTerrain3DExaggeration } from 'reducers/map/selectors';
```
Comment (anisometropie) [inline front/src/common/Map/Layers/HillShade.tsx]: Hillshade is just one word, no need to rename that file

**Review by anisometropie — CHANGES_REQUESTED** [review]

File: front/src/common/Map/Layers/HillShade.tsx (hunk 1 above)
Comment (clarani) [reply front/src/common/Map/Layers/HillShade.tsx]: Done :white_check_mark:

**Review by Synar — APPROVED** [review]
LGTM!

**Review by Math-R — APPROVED** [review]
lgtm !

**Review by anisometropie — APPROVED** [review]

**Pull request merged**